lib/msal-common/package.json

```diff
@@ -10,7 +10,7 @@
     "type": "git",
     "url": "https://github.com/AzureAD/microsoft-authentication-library-for-js.git"
   },
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
@pkanher617 Should this be bumped manually or does beachball take care of that?

Reply:
I'm not sure why the version is being bumped in this PR, this should only be done before a release.

Reply:
@pkanher617 Ok, thanks. I see the build fails for me anyways, so I will open an issue and abandon this PR instead.